### ISSUE TITLE
Add initial-version configuration option

### ISSUE THREAD

**Issue by baolsen** [google-github-actions/release-please-action#683]
### TL;DR

Quote from the [latest Semantic Versioning specification](https://semver.org/)

> **How should I deal with revisions in the 0.y.z initial development phase?**
> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

It is becoming more commonplace to see the convention of 0.x.y as pre-releases under active development, and 1.0.0 as a "stable production release".

To allow this convention, we would like to be able to set initial version to 0.1.0 on the github action and have this honoured by the release process. 

### Detailed design

Something like this would be great.

```
    steps:
      - name: release-please
        uses: google-github-actions/release-please-action@v3
        id: release
        with:
          release-type: simple
          initial-release: v0.1.0
          # BREAKING CHANGE commits only bump semver minor level, if version < 1.0.0
          bump-minor-pre-major: true
          # feat commits bump semver patch level instead of minor, if version < 1.0.0
          bump-patch-for-minor-pre-major: true
```

### Additional information

- release-please seems to [implement this](https://github.com/googleapis/release-please/search?q=initialVersion) in some way.

- Related to #425 . 
When 0.x.y is the current version we should be sure to respect the bump settings: 
`bump-minor-pre-major: true`and `bump-patch-for-minor-pre-major: true`

**Comment by baolsen** [comment]
Note: There is a way to do this by adding `Release-As: 0.1.0` in commit message. But this is an easily forgotten manual step.

**Comment by bcoe** [comment]
@baolsen I'm open to this feature if you'd like to send a patch.

**Comment by kcirtapfromspace** [comment]
+1 Yeah, this is a bit hidden from a just getting started dev phase and cumbersome to track down for any new project. 

Right off the FAQ:
https://semver.org/

> Major version zero is all about rapid development. If you’re changing the API every day you should either still be in version 0.y.z or on a separate development branch working on the next major version.

**Comment by chingor13** [comment]
Apologies, but for maintainability, we cannot continue to pass through every configuration option in the form of action inputs.

In v4, we are removing most configuration options which can be set in the manifest configuration file. This will make updating the action with the latest version/features easier and allow folks to debug their configuration using the `release-please` CLI.
